CI/travis.before_install.sh

```diff
@@ -1,8 +1,4 @@
 #!/bin/bash
-if [ -z "${TRAVIS_OS_NAME}" ] || [ "${TRAVIS_OS_NAME}" = "linux" ]; then
-  echo Before install on linux.
```

Comment:
... IIRC I even set the `qmake`/`cmake` makefile producers,  and the selected compiler report their versions before they got down to business - again this is just to help pin down details if something goes wrong...

_just because you are paranoid, it does __not__ mean the system is __not__ out ot get you_